napari/plugins/_npe2.py

```diff
@@ -282,11 +296,105 @@ def _on_plugin_enablement_change(enabled: Set[str], disabled: Set[str]):
     to_disable.update(disabled)
     plugin_settings.disabled_plugins = to_disable
 
+    for plugin_name in enabled:
+        # technically, you can enable (i.e. "undisable") a plugin that isn't
+        # currently registered/available.  So we check to make sure this is
+        # actually a registered plugin.
+        if plugin_name in pm.instance():
+            _register_manifest_actions(pm.get_manifest(plugin_name))
+
+    # TODO: after app-model, these QMenus will be evented and self-updating
```

Comment:
Can/ should this ToDo be addressed before merge?

Reply:
nope, that's out of scope here, and requires all of the other menu prs firs (#4826, #4865, etc...)